benchopt/util.py

```diff
@@ -290,28 +287,28 @@ def is_included(name, include_patterns=None):
     return False
 
 
-def create_venv(env_name, recreate=False):
-    """Create a virtual env with name env_name and install basic utilities"""
+def create_condaenv(env_name, recreate=False):
+    """Create a conda env with name env_name and install basic utilities"""
 
-    env_dir = f"{VENV_DIR}/{env_name}"
+    force = " -y" * recreate
```

Comment:
Not sure if I like this syntax. Maybe putting `" y" if recreate else ""` is more explicit/maintanable.